.github/workflows/shared-ci-terraform.yml

```
@@ -176,3 +176,12 @@ jobs:
           working_directory: ./target/${{ matrix.tfdir }}
           filter_mode: ${{ inputs.filter-mode }}
           tflint_version: "v0.49.0"
+
+  test:
+    if: ${{ github.event_name == 'pull_request' && startsWith(github.head_ref, 'mergify/merge-queue/') }}
+    uses: cloudposse/.github/.github/workflows/shared-terratest.yml@merge-queue
+    needs: [lint]
+    with:
+      pr_number: ${{ github.event.pull_request.number }}
+      runs-on: ${{ inputs.runs-on }}
+            
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)